server/routes/relatedProductsHelper.js

```diff
@@ -46,23 +45,69 @@ const getProductID = (productId) => {
 };
 
 const getProductStyles = (productId) => {
-  return new Promise((resolve, reject) => {
-    axios({
-      method: 'GET',
-      url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/products/${productId}/styles`,
-      headers: {
-        'User-Agent': 'request',
-        'Authorization': config.TOKEN
+
+
+  let proms = [];
+  let users = [];
+
+  return axios({
+    method: 'GET',
+    url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/products/${productId}/related`,
+    headers: {
+      'User-Agent': 'request',
+      'Authorization': config.TOKEN
+    }
+  })
+    .then((data) => {
+      for (let i = 0; i < data.data.length; i++) {
+        let relatedIds = data.data[i];
+        proms.push(
+          axios({
+            method: 'GET',
+            url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/products/${relatedIds}/styles`,
+            headers: {
+              'User-Agent': 'request',
+              'Authorization': config.TOKEN
+            }
+          })
+
+            .then((relatedInfo) => {
+              return users.push(relatedInfo.data);
+              // console.log('HERE==', relatedInfo.data)
+            }))
       }
+
+      return Promise.all(proms)
+        .then(() => {
+          return users;
+        })
+
     })
-      .then((data) => {
-        resolve(data.data);
-      })
-      .catch((err) => {
-        reject(err);
-      })
-  })
-}
+    .catch((err) => {
+      console.log(err);
+    })
+
+
+  // return new Promise((resolve, reject) => {
```

Comment:
You have a few spots like this with commented out chunks of code? Do you still need these? If so, definitely keep it in! But it not, you could delete it for better optimization.